src/components/navs/AppNav/AppNavSettings.vue

```diff
@@ -221,6 +221,7 @@ export default defineComponent({
     const connectorName = computed(() => getConnectorName(connector.value.id));
 
     const connectorLogo = computed(() => getConnectorLogo(connector.value.id));
+    const hideDisconnect = computed(() => connector.value.id == 'gnosis');
```

Comment:
I've seen that in other instances you have used
```
connector.value?.id === 'gnosis'
```

should this be the case here too?

Reply:
It's not really necessary as I think these components don't get rendered until we have a connector (In other instances I was getting crashes due to trying to read from a null connector.) It doesn't hurt though.